File: C/gob.c
#include <stdio.h>

int main(int argc, char* argv[]) {
	
	char op;
	printf("Gob's program? y/n\n");
	op = getc(stdin);
	
	if ((char)op != 'y')
		return 0;
	
	char m[] = "Penus ";
	char* p;

	for (p = m; /* derp */ ; p = m) while (*p != '\x00') putc(*p++, stdout);
	return 0;
}

File: nodejs/gob.js
process.stdout.write('Gob\'s Program Y/N?\n');
process.stdout.write('?');

process.stdin.resume();
process.stdin.setEncoding('utf8');

process.stdin.on('data', function (michael) {
    if ( michael === 'Y\n' ) {
        while ( true ) {
            process.stdout.write("Penus ");
        }
    } else {
        process.exit(1);
    }
});
